u05-sa-logistic-regression/answer.md

```diff
@@ -0,0 +1 @@
+In this step, first the logarithm of the softmax results is calculated. Then these probabilities are multiplied by the corresponding labels in the training dataset, and then summed. This calculates the log-likelihood of the model. 
```

Comment:
What does multiplying by corresponding labels do (you're thinking one-hot encoded?)?